### PR TITLE
change min amount to 0 and staking token name, added wallet to platfo…

### LINKED ISSUE #9347: ankr staking info issues
### Describe the bug

there is incorrect data about ankr liquid staking here: https://ethereum.org/en/staking/pools/#explore-staking-pools
we have ETH staking and unstaking for any amount, can be found here: https://www.ankr.com/staking/stake/ethereum/
we have staking avaivable though Clover wallet: https://wallet.clover.finance/
naming for staking token has been changed: https://etherscan.io/address/0xE95A203B1a91a908F9B9CE46459d101078c2c3cb

### To reproduce

1. open page https://ethereum.org/en/staking/pools/#explore-staking-pools
2. you can see FROM 0.5 ETH
3. you can see that "Wallet" badge is missing

### Expected behavior

Insteed of "FROM 0.5 ETH" is should say "Any amount"
"Wallet" badge should be visible

### Screenshots

image

### Desktop (please complete the following information)

No response

### Smartphone (please complete the following information)

No response

### Additional context

No response

### PR DESCRIPTION
**Pull request by mikrowelt**
…rms 

[Fixes #9347]

change min amount to 0 and staking token name, added wallet to platforms

### CODE REVIEW

**Comment by le0n229** [comment]
Hello, can you please check and merge it? @jmcook1186

**Review by hoanglamcoin — APPROVED** [review]

**Review by corwintines — APPROVED** [review]
Thanks @mikrowelt

**Comment by corwintines** [comment]
@wackerow do you have a moment to review as well?

File: src/data/staking-products.json [hunk 1]
```diff
@@ -660,8 +660,8 @@
           "address": "0xD01ef7C0A5d8c432fc2d1a85c66cF2327362E5C6"
         },
         {
-          "name": "Ankr Eth2 Reward Bearing Certificate",
```
Comment (wackerow) [inline src/data/staking-products.json:663]: :heart: Goodbye Eth2 =)

**Review by wackerow — APPROVED** [review]
LGTM!

**Pull request merged**